Description: Remove requirement for bundler/setup
Author: Jackson Isaac<jacksonisaac2008@gmail.com>
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=714242
Last-Update: 2013-08-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/helper.rb
+++ b/test/helper.rb
@@ -1,4 +1,3 @@
-require 'bundler/setup'
 require 'minitest/spec'
 require 'minitest/autorun'
 require 'active_record'
